evergreen.py

```diff
@@ -80,8 +80,8 @@ def main():  # pragma: no cover
             print("Skipping " + repo.full_name + " (visibility-filtered)")
             continue
         existing_config = None
-        filename_list = [".github/dependabot.yml", ".github/dependabot.yaml"]
-        dependabot_filename_to_use = None
+        filename_list = [".github/dependabot.yaml", ".github/dependabot.yml"]
```

Comment:
I'm good with this idea but why flip the array order?

Reply:
It is the recommended extension AFAIK, with `yml` being sort of legacy.

https://yaml.org/faq.html

But it doesn't really matter if you rather not have the list flipped I can remove this part.

Reply:
> It is the recommended extension AFAIK, with `yml` being sort of legacy.
> 
> 
> 
> https://yaml.org/faq.html
> 
> 
> 
> But it doesn't really matter if you rather not have the list flipped I can remove this part.

Cool. Was just curious. Both are detected. Thanks.